hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_order_book.py

```diff
@@ -1,43 +1,67 @@
+from dataclasses import dataclass
 from typing import Dict, List, Optional
 
-from bxsolana_trader_proto import GetOrderbookResponse
+from bxsolana_trader_proto import GetOrderbookResponse, api
 from bxsolana_trader_proto.api import OrderbookItem
 
+from hummingbot.connector.exchange.bloxroute_openbook.bloxroute_openbook_utils import truncate
 from hummingbot.core.data_type.order_book import OrderBook
-from hummingbot.core.data_type.order_book_message import OrderBookMessage, OrderBookMessageType
 from hummingbot.core.data_type.order_book_row import OrderBookRow
 
 
+@dataclass
+class Orderbook:
+    asks: List[api.OrderbookItem]
+    bids: List[api.OrderbookItem]
+
+
+@dataclass
+class OrderbookInfo:
+    best_ask_price: float
+    best_ask_size: float
+    best_bid_price: float
+    best_bid_size: float
+    latest_order_book: Orderbook
+    timestamp: float
+
+
+@dataclass
+class OrderStatusInfo:
+    order_status: api.OrderStatus
+    quantity_released: float
+    quantity_remaining: float
+    side: api.Side
+    fill_price: float
+    order_price: float
+    client_order_i_d: int
+    timestamp: float
+
+    def __eq__(self, other: "OrderStatusInfo"):
```

Comment:
I think this is unnecessary since `@dataclass` generates an `__eq__` implementation